pkg/docker/docker-entrypoint.sh

```diff
@@ -1,11 +1,11 @@
-#!/usr/bin/env bash
```

Comment:
The shebang for sh(1) should be  `#!/bin/sh`.

Reply:
done

Reply:
but wouldnt it make it less portable?

Reply:
I think not.  In case of `bash(1)`, yes, because some systems install `bash(1)` in a weird place, so `env(1)` always knows where it is.

However, `sh(1)` is always (AFAIK) available at `/bin/sh`.

I just looked at POSIX, and it seems that it allows `sh(1)` to be in a different location.  But that would break lots of scripts, so I don't think any reasonable system puts `sh(1)` in a different place.  BTW, I guess there's a reason most sources refer to `sh(1)` as `/bin/sh` instead of `sh(1)`.

Reply:
thanks for the info